src/Components/Registeration/base.js

```diff
@@ -0,0 +1,13 @@
+import * as firebase from "firebase/app";
+import "firebase/auth";
+
+const app = firebase.initializeApp({
+  apiKey: "AIzaSyB-yIaoqsFya7T5PWiy6nAVKONXqRz1VOk",
```

Comment:
Hey the apiKey, I'm not super familiar with firebase why is that not an env variable as well for security.